packages/nimble-components/src/combobox/specs/IxD.md

```diff
@@ -0,0 +1,241 @@
+| Label           | Text that tells the end-user what to expect in the component                |
+| Field input     | Input that displays the current selected or entered option                  |
+| Dropdown button | Click area that opens the dropdown                                          |
+| Dropdown        | A list of options to choose from displayed as an _open_ state               |
+| List item       | A selection the end-user can make, shown with other options in the dropdown |
+
+#### Label
+
+Comboboxes should always have a label, unless approved by an interaction designer or user researcher (usually in toolbar use cases). _See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
+
+#### Field Input
+
+Comboboxes can be empty by default or have a default selection from the menu. Empty comboboxes should include placeholder text that propts the end-user to enter a custom value or select an option from the menu. Placeholder text should always follow the pattern "Enter or select [thing(s)]", for example "Enter or select address". Ellipses are not needed for the placeholder. Use sentence casing for the placeholder.
+
+![ ](spec-images/combobox-2.png)
+
+#### Dropdown & List Items
+
+A list of options to select, displayed as an open state._See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
+
+### Related Components
+
+To allow end-users to select a list item but not enter a custom value, use a [**select**](/packages/nimble-components/src/select/specs/IxD.md).
+
+![ ](spec-images/combobox-alternative-select.png)
+
+To allow end-users to multi-select list items, use a [**tag picker**](/packages/nimble-components/src/tag-picker/specs/IxD.md).
+
+![ ](spec-images/combobox-alternative-picker.png)
+
+To allow end-users to select less than three options, use a [**radio group**](/packages/nimble-components/src/radio-group/specs/IxD.md).
+
+## Design
+
+### Configuration
+
+#### Label
+
+The client-user should be able to specify where they want the label to be positioned. _See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
+
+#### Icons
+
+The client-user should be able to specify whether they want left icons on list items or not. _See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
+
+#### Groups
+
+The client-user should be able to create groups for list items. _See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
+
+> NOTE: NEEDS VISUAL DESIGN
+
+The client should be able to specify whether they want the group name to be visible on the component when the dropdown is closed.
+
+![ ](spec-images/combobox-26.png)
+
+#### Secondary Text
+
+The client-user should be able to specify whether they want secondary text on list items or not. _See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
```

Comment:
We could show secondary text in the options list, but similar to above there is no selection of an option, just autocompletion of text. so secondary text / group name would not appear in the text box of the combobox.